src/certificate_provider.py

```diff
@@ -89,7 +95,7 @@ def update(self):
                     self.fail("{}".format(error))
             elif changed_properties:
                 self.fail(
-                    'You can only change the "Options" and "DomainName" '
```

Comment:
@mvanholsteijn SubjectAlternativeNames is not supported currently. This PR will support it:
https://github.com/binxio/cfn-certificate-provider/pull/24

https://docs.aws.amazon.com/acm/latest/APIReference/API_RequestCertificate.html#ACM-RequestCertificate-request-SubjectAlternativeNames